sim/src/test/scala/bridges/TracerVSuite.scala

```diff
@@ -0,0 +1,61 @@
+//See LICENSE for license details.
+
+package firesim.bridges
+
+import java.io._
+
+import org.scalatest.Suites
+import org.scalatest.matchers.should._
+
+import freechips.rocketchip.config.Config
+import freechips.rocketchip.config.Parameters
+import freechips.rocketchip.config._
+
+import firesim.{BasePlatformConfig, TestSuiteCommon}
+
+abstract class TracerVTestBase(
+  platformConfig: BasePlatformConfig,
+  width:          Int,
+  trace:          Option[Int]    = None,
+  start:          Option[String] = None,
+) extends BridgeSuite("TracerVModule", s"TracerVModuleTestCount${width}", platformConfig) {
+  override def runTest(backend: String, debug: Boolean) {
+    // Create an expected file.
+    val expected = File.createTempFile("expected", ".txt")
+    expected.deleteOnExit()
+
+    // Create the output file. tracerv will always append '-C0' to the end of the path provided in the plusarg
+    val output     = File.createTempFile("output", ".txt-C0")
+    output.deleteOnExit()
+    val outputPath = output.getPath.substring(0, output.getPath.length() - 3)
```

Comment:
Hoist the suffix out into it's own `val` and use [stripSuffix](https://www.scala-lang.org/api/2.12.7/scala/collection/immutable/StringLike.html#stripSuffix(suffix:String):String)